modules/indexing/src/test/java/org/apache/ignite/client/ClientTestSuite.java

```diff
@@ -47,6 +47,7 @@
 import org.apache.ignite.internal.client.thin.events.IgniteClientConnectionEventListenerTest;
 import org.apache.ignite.internal.client.thin.events.IgniteClientLifecycleEventListenerTest;
 import org.apache.ignite.internal.client.thin.events.IgniteClientRequestEventListenerTest;
+import org.apache.ignite.sqltests.ExtraColumnInH2RowsTest;
```

Comment:
This test suite related to thin client tests. So, either test should be in correspoinding thin client package, or should be moved to another suite (related to SQL tests).

Reply:
Test moved to `org.apache.ignite.client.thin`

Reply:
It was copied, and now there are two tests with name ExtraColumnInH2RowsTest. Old one must be removed.